docs/manuals/man1/pki.1.md

```diff
@@ -0,0 +1,350 @@
+  
+CLI OPTIONS
+-----------
+
+`-c` &lt;NSS database password&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Specifies the NSS database password (mutually exclusive to the `-C` option).
+
+`-C` &lt;NSS database password file&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Specifies the file which contains the NSS database password (mutually exclusive to the `-c` option).
+
+`-d` &lt;NSS database location&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Specifies the NSS database location (default: *&#8764;/.dogtag/nssdb*).
+
+`-h` &lt;hostname&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Specifies the hostname (default: hostname of the local machine).
+
+`--help`  
+&nbsp;&nbsp;&nbsp;&nbsp;Prints additional help information.
+
+`--ignore-cert-status` &lt;list&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Comma-separated list of ignored certificate validity statuses.
+
+`--message-format` &lt;format&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Message format: xml (default), json.
+
+`-n` &lt;client certificate nickname&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Specifies the nickname for client certificate authentication (mutually exclusive to the `-u` option).
+
+`--output` &lt;folder&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Folder to store HTTP messages.
+
+`-P` &lt;protocol&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Specifies the protocol (default: http).
+
+`-p` &lt;port&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Specifies the port (default: 8080).
+
+`--reject-cert-status` &lt;list&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Comma-separated list of rejected certificate validity statuses.
+
+`-t` &lt;type&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Subsystem type.
+
+`--token` &lt;token&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Security token name
+
+`-U` &lt;URL&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Specifies the server URL.
+
+`-u` &lt;username&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Specifies the username for basic authentication (mutually exclusive to the `-n` option).
+
+`-v`, `--verbose`  
+&nbsp;&nbsp;&nbsp;&nbsp;Displays verbose information.
+
+`--version`  
+&nbsp;&nbsp;&nbsp;&nbsp;Displays CLI version information.
+
+`-w` &lt;password&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Specifies the user password (mutually exclusive to the `-W` option).
+
+`-W` &lt;client-side password file&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Specifies the file which contains the user password (mutually exclusive to the `-w` option).
+
+OPERATIONS
+----------
+
+To view available commands and options, simply type `pki`.  Some commands have sub-commands.
+To view the sub-commands, type `pki <command>`.
+To view each command's usage, type `pki <command> --help`.
+
+An NSS database is needed to execute commands that require crypto operations such as establishing SSL connection.
+See `pki-client`(1) for more information.
+
+### Connection
+
+By default, the CLI connects to a server running on the local machine via the non-secure HTTP port 8080.
+To specify a different server location, use the appropriate arguments to give a different host (`-h`), port (`-p`), or connection protocol (`-P`).
+
+```
+$ pki -P <protocol> -h <hostname> -p <port> <command>
```

Comment:
I'm actually intentionally use $ prompt to distinguish the command that the user needs to type from the output of the command (sometimes we show what the output looks like). Without the prompt it could be confusing. Also, I'd rather not use # as a prompt since it could be confused with comments. Besides, the pki CLI can be used by either a normal user or root user, so the $ prompt is a general purpose prompt, and the user can add sudo as needed.